config.yaml

```
@@ -1,14 +1,16 @@
-# Copyright 2022 Canonical Ltd.
-# See LICENSE file for licensing details.
-#
-# TEMPLATE-TODO: change this example to suit your needs.
-# If you don't need a config, you can remove the file entirely.
-# It ties in to the example _on_config_changed handler in src/charm.py
-#
-# Learn more about config at: https://juju.is/docs/sdk/config
-
 options:
-  thing:
-    default: 🎁
-    description: A thing used by the charm.
+  multitenant_label:
     type: string
+    description: |
+      Alertmanager Configurer has been designed to support multiple tenants. In a multitenant 
+      Alertmanager Configurer setup, each alert is first routed on the tenancy label, and then 
+      the routing tree is distinct for each tenant.
+    default: networkID
```

```suggestion
    default: ""
```
We probably don't want networkID as a default as it is magma specific.

---

Done.